index.html

```diff
@@ -108,63 +128,74 @@ <h4>Profesional Art Printing Data</h4>
+          </div>
+            <ul>
+              <li class="aboutme-frame-item">Bootstrap</li>
+              <li class="aboutme-frame-item">Ruby on Rails</li>
+              <li class="aboutme-frame-item">RSPec</li>
+              <li class="aboutme-frame-item">CapyBara</li>
+              <li class="aboutme-frame-item">Elenium</li>
+            </ul>
         </div>
-          <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>RSPec</li>
-            <li>CapyBara</li>
-            <li>Elenium</li>
-          </ul>
-        <div class="all">
-          <img src="images/Circle.png" alt="circle-image">
-          <h5>Skills</h5>
+        <div class="aboutme-Languages">
+          <div class="all">
+            <img src="images/Circle.png" alt="circle-image">
+            <h5>Skills</h5>
+          </div>
+            <ul>
+              <li class="aboutme-skills-item">Codekit</li>
+              <li class="aboutme-skills-item">Guthub</li>
+              <li class="aboutme-skills-item">Codepen</li>
+              <li class="aboutme-skills-item">Gitlab</li>
+              <li class="aboutme-skills-item">Terminal</li>
+            </ul>
         </div>
-          <ul>
-            <li>Codekit</li>
-            <li>Guthub</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
       </div>
     </section>
     <section class="contact-form">
-      <p>
-        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-      </p>
-      <form action="https://formspree.io/f/mnqwdall" method="post">
-        <input class="contact-name" type="text" name="full name" id="signup" maxlength="30" placeholder="Full name" required>
-        <hr class="form-line">
-        <input class="contact-email" type="email" name="email" id="fillup" placeholder="Email address" required>
-        <hr class="form-line">
-        <textarea name="msg" id="textup" cols="25" rows="4" maxlength="500" placeholder="Enter text here" required></textarea>
-        <input type="submit" class="contact-button" value="Get in touch"/>
-      </form>
+      <div class="contact-section_form">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </p>
+        <form action="https://formspree.io/f/xeqngkaa" method="post">
+          <input class="contact-name" type="text" name="full name" id="signup" maxlength="30" placeholder="Full name" required>
+          <hr class="form-line">
+          <input class="contact-email" type="email" name="email" id="fillup" placeholder="Email address" pattern="[Ee]mail" required>
+          <hr class="form-line">
+          <textarea name="msg" id="textup" cols="25" rows="4" maxlength="500" placeholder="Enter text here" required></textarea>
+          <input type="button" class="contact-button" value="Get in touch"/>
```

Comment:
- [ ] [REQUIRED] Your form submission still doesn't work. Kindly change the type of the input from `button` to `submit`.
- [ ] Please make sure that the form is submitted successfully before requesting another review. You should see this message after submitting the form:
<img width="223" alt="image" src="https://user-images.githubusercontent.com/77190203/154380297-0a2dcafe-ec4d-4d5d-957c-25d9973395fb.png">
- [ ] Kindly work more on the design of your contact form. It would be better to use the same margin for all the inputs of the form to improve the UI design.
<img width="890" alt="image" src="https://user-images.githubusercontent.com/77190203/154380001-31cd6c6c-9705-4eb2-bc1d-1a74d77e8471.png">